components/LinkCustom/LinkCustom.tsx

```diff
@@ -19,17 +18,13 @@ interface ILinkCustomProps {
  * @param props.className - any additional classes you want to add to that instance of the LinkCustom component.
  * @returns The custom link component
  */
-const LinkCustom = ({
-  children,
-  className,
-  href,
-}: ILinkCustomProps): JSX.Element => {
+const LinkCustom = ({ children, href, ... props }: ILinkCustomProps): JSX.Element => {
   return (
     <Link
-      className={cn('hover:text-orange-600 hover:underline', className)}
-      data-testid="linkCustom"
+      className={'font-bold text-orange-600 hover:text-orange-600 hover:underline'}
```

Comment:
```suggestion
      className={'font-bold text-orange-600 hover:text-orange-600 hover:underline focus:text-orange-600 focus:underline'}
```

Reply:
@ryandotfurrer 
Are you in favor of the underline over the default focus outline or you'd like to have both?
Just the underline would not be enough. Especially because most links are underlined in default state, a sighted keyboard user might not necessarily be able to tell that the link is in focus.

The default (and browser-specific) focus outline is indeed boring. Should we ask Jayna for custom-made focus style?

Reply:
@CorinaMurg we discussed on last Thursday's stream (IIRC) that we wanted to make our `focus` styles the same as our `hover` styles. I would elect to have both the default focus outline _and_ the underline.

@shashilo do you recall this conversation?

Reply:
@ryandotfurrer 
Sorry, I missed that part. Was it just for links? If it's for all interactive elements, maybe it would be better to have a separate ticket?

Reply:
@CorinaMurg We discussed it when talking about the `WeeklyPickButton`, however, I mentioned how I had neglected it on every component I had created thus far: which includes `LinkCustom.`

While a ticket would be wise to ensure _all_ components with a `hover` state also have `focus` states applied, since you're already in here I thought I'd mention it.

@shashilo, how would you like Corina to proceed?